connectors/config.py

```diff
@@ -60,9 +64,12 @@ def _default_config():
                 "chunk_size": 1000,
                 "max_concurrency": 5,
                 "chunk_max_mem_size": 5,
+                "max_retries": 5,
+                "retry_interval": 10,
```

Comment:
nit: These two could use the constants, too.

Reply:
Addressed in 244ce2e7e29534bf2c08507e5fa59f496524e228